django/contrib/admin/options.py

```diff
@@ -327,6 +327,12 @@ def get_fieldsets(self, request, obj=None):
             return self.fieldsets
         return [(None, {'fields': self.get_fields(request, obj)})]
 
+    def get_inlines(self, request, obj=None):
```

Comment:
It isn't used, yes, but compare to `get_exclude` which also doesn't use it in the default implementation.

Having it allows overriding implementations to choose the inlines based on the instance. Not having it may well result in a future request to add it. It also neatly propagates down from `get_inline_instances()`.

Reply:
Sorry for being unclear. I was talking about `None` as a default value for `obj`:

```python
def get_inlines(self, request, obj):
```